nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/DirectoryTest.java

```diff
@@ -395,6 +396,29 @@ public void itCanQueryDirectoryEntryWithIdContainingSlashes() throws Exception {
                 node.get("properties").get("label").asText());
     }
 
+    @Test
+    public void itReturnsProperPagination() throws Exception {
+        // Given a directory modified entry as Jsont
```

Comment:
Jsont -> Json

If you use a "Given" comment then you should follow the rest of the pattern: have for instance a "When I do this" comment and a "I get this result" comment.